### PR TITLE
release of psmt2-frontend 0.4.0

### PR DESCRIPTION
**Pull request by ACoquereau**

### CODE REVIEW

File: packages/psmt2-frontend/psmt2-frontend.0.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ]
+]
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "dune" {>= "2.6.0"}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {with-test}
+  "menhir" {}
```
Comment (mseri) [inline packages/psmt2-frontend/psmt2-frontend.0.4.0/opam:35]: Do you know if this requires a specific bound?

Comment (ACoquereau) [reply packages/psmt2-frontend/psmt2-frontend.0.4.0/opam:35]: Not that i know of but i will try to test older version of menhir. I'll keep you informed

Comment (ACoquereau) [reply packages/psmt2-frontend/psmt2-frontend.0.4.0/opam]: I was unable to compile the project with a version of menhir older than `20180528`. I've update the constraint, is that ok for you ?

Comment (mseri) [reply packages/psmt2-frontend/psmt2-frontend.0.4.0/opam]: Perfect, thanks you very much!

**Pull request merged**